### ISSUE TITLE
t/test_archive_libarchive.t fails (with older Test2::API?)

### ISSUE THREAD

**Issue by eserte** [uperl/Test-Archive-Libarchive#2]
I see on some systems the following test failure:
```
    # Failed test 'missing method'
    # at t/test_archive_libarchive.t line 36.
    # +----------------+-----+----------------+----+----------------+--------+
    # | PATH           | LNs | GOT            | OP | CHECK          | LNs    |
    # +----------------+-----+----------------+----+----------------+--------+
    # |                |     | ARRAY(0x802d44 |    | <ARRAY>        | 32, 35 |
    # |                |     | 4c8)           |    |                |        |
    # |                |     |                |    |                |        |
    # | [0]            | 31  | Test2::Event:: |    | <EVENT: Fail>  | 33, 33 |
    # |                |     | Ok=HASH(0x801c |    |                |        |
    # |                |     | c48b8)         |    |                |        |
    # |                |     |                |    |                |        |
    # | [0] <blessed>  |     | Test2::Event:: | eq | Test2::Event:: |        |
    # |                |     | Ok             |    | Fail           |        |
    # +----------------+-----+----------------+----+----------------+--------+
    # Failed test 'bad return'
    # at t/test_archive_libarchive.t line 47.
... etc ...
    # Failed test 'bad return'
    # at t/test_archive_libarchive.t line 57.
    # +----------------+-----+----------------+----+----------------+--------+
    # | PATH           | LNs | GOT            | OP | CHECK          | LNs    |
    # +----------------+-----+----------------+----+----------------+--------+
    # |                |     | ARRAY(0x803bf9 |    | <ARRAY>        | 53, 56 |
    # |                |     | a38)           |    |                |        |
    # |                |     |                |    |                |        |
    # | [0]            | 52  | Test2::Event:: |    | <EVENT: Fail>  | 54, 54 |
    # |                |     | Ok=HASH(0x8040 |    |                |        |
    # |                |     | 56c90)         |    |                |        |
    # |                |     |                |    |                |        |
    # | [0] <blessed>  |     | Test2::Event:: | eq | Test2::Event:: |        |
    # |                |     | Ok             |    | Fail           |        |
    # +----------------+-----+----------------+----+----------------+--------+

# Failed test 'la_fatal'
# at t/test_archive_libarchive.t line 60.
t/test_archive_libarchive.t .. 
Dubious, test returned 5 (wstat 1280, 0x500)
Failed 5/6 subtests 
```

Statistical analysis suggests that this happens for older Test2::API (e.g. 1.302162):
```
****************************************************************
Regression 'mod:Test2::API'
****************************************************************
Name           	       Theta	      StdErr	 T-stat
[0='const']    	      0.0000	      0.0000	   1.48
[1='eq_1.302162']	      0.0000	      0.0000	   4.17
[2='eq_1.302164']	      1.0000	      0.0000	4431066882818686.50
[3='eq_1.302183']	      1.0000	      0.0000	8462129931575686.00
[4='eq_1.302185']	      1.0000	      0.0000	8523327329397027.00

R^2= 1.000, N= 75, K= 5
****************************************************************
```

**Comment by plicease** [comment]
This is an incompatibility with older versions of Test2::Suite, and the bug is in this module.

**Comment by plicease** [comment]
This should be fixed in 0.02
